### DISCUSSION TITLE
Derelict sensor entities safe to delete?

### DISCUSSION THREAD

**Discussion by fhteagle** [firstof9/openevse#141] [Q&A] [answered]
Upgraded to 2.1.8. Noticed sensor.openevse_wifi_firmware_version is listed with a status of "restored" on the entities list. There is a also a sensor.openevse_wifi_firmware_version_2 listed on this screen. 

Can you verify that it is safe to delete the old sensor.openevse_wifi_firmware_version, sensor.openevse_wifi_signal_strength , and button.openevse_restart entities?

Thanks.

**Comment by firstof9** [comment, marked as answer]
You're safe to remove entities that say they're no longer supplied by the integration.
